CMakeLists.txt

```diff
@@ -256,6 +266,9 @@ project (Jerry CXX C ASM)
     # stm32f4
      set(SOURCE_JERRY_STANDALONE_MAIN_MCU_STM32F4 main-mcu.cpp)
 
+   # EXTERNAL dummy
+    set(SOURCE_JERRY_STANDALONE_MAIN_EXTERNAL main-external.cpp)
```

Comment:
@seanshpark, maybe we shouldn't add the file, as external application provides its own `main`. So, jerryscript, in the case, is just linked to some another application.